server/services/v1/search.go

```diff
@@ -120,7 +121,8 @@ func (s *searchService) DeleteIndex(ctx context.Context, req *api.DeleteIndexReq
 	runner.SetDeleteIndexReq(req)
 
 	resp, err := s.sessions.TxExecute(ctx, runner, search.SessionOptions{
-		IncVersion: true,
+		IncVersion:          false,
```

Comment:
We need to make code change for this. Should we just have a single flag for this?